pkgs/applications/office/paper-note/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, cmake
+, pkg-config
+, python3
+, vala
+, glib
+, gtk3
+, gtk4
+, libgee
+, libadwaita
+, gtksourceview5
+, blueprint-compiler
+, wrapGAppsHook4
+, appstream-glib
+, desktop-file-utils
+}:
+
+stdenv.mkDerivation rec {
+  pname = "paper-note";
+  version = "22.11";
+
+  src = fetchFromGitLab {
+    owner = "posidon_software";
+    repo = "paper";
+    rev = version;
+    hash = "sha256-o5MYagflHE8Aup8CbqauRBrdt3TrSlffs35psYT7hyE=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    vala
+    pkg-config
+    wrapGAppsHook4
+    appstream-glib
+    desktop-file-utils
+  ];
+
+  buildInputs = [
+    cmake
```

Comment:
I don't see anything obvious which requires cmake in the source repository for this app. Does this need to be included? It's unusual to need both meson and cmake (for a project that simple at least).

If so, it should likely be in nativeBuildInputs.

Reply:
I guess this is https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/meson-cmake.md, and can be removed as long as the package builds :thinking:

Reply:
builds without cmake buildinput!